lms/djangoapps/email_marketing/tasks.py

```diff
@@ -294,6 +296,72 @@ def _get_course_content(course_url, sailthru_client, email_config):
     return response
 
 
+def _get_or_create_user_lists(sailthru_client, site_domain=None):
+    """
+    Get the user list name from cache if exists else create one and return the name
+    :param sailthru_client:
+    :param user_lists:
+    :return: list name if exists or created else return None
+    """
+    email_config = EmailMarketingConfiguration.current()
+    if not email_config.enabled:
+        return
+
+    if site_domain and "edx.org" not in site_domain:
+        list_name = site_domain.replace(".", "_") + "_user_list"
+    else:
+        list_name = "All edX Users"
```

Comment:
`list_name = _format_sailthru_list_name(site.domain) if site and site.id != settings.SITE_ID else default_list_name`

Add the `_format_sailthru_list_name(site_domain)` function:
```
def _format_sailthru_list_name(site_domain):
    return site_domain.replace(".", "_") + "_user_list"
```